rules-tests/Renaming/Rector/Name/RenameClassRector/FixtureAutoImportNames/auto_import_conflict_name_all_fqcn_same_namespace_inside.php.inc

```diff
@@ -1,33 +0,0 @@
-<?php
-
-namespace Foo2;
-
-class AutoImportConflictNameAllFqcnSameNamespaceInside extends FooBar
```

Comment:
why fixture removed? what is new behavior?

Reply:
3 fixture removed, 2 renamed, so there is 1 fixture got deleted, what new behaviour?